example.secrets.dev.yml

```diff
@@ -12,4 +12,6 @@ SLACK_VERIFICATION_TOKEN: null
 PD_API_URL: https://apis.paralleldots.com
 
 # Go get a token here https://www.paralleldots.com/text-analysis-apis
-PD_API_KEY: null
\ No newline at end of file
+PD_API_KEY: null
+
+DYNAMODB_TABLE_PREFIX: "dont-slack-evil-dev-messages"
```

Comment:
```suggestion
DYNAMODB_TABLE_PREFIX: "dont-slack-evil-dev-test"
```

Reply:
Non justement le serverless.dev.yml c'est la prod (enfin quand on déploie en local) ! donc on laisse messages

Reply:
I do not understand why would "message" would be in the prefix?

Reply:
Sorry I'm bling =_=